<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyDialog</class>
 <widget class="QDialog" name="CopyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>380</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Translation</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTranslation">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>380</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Starting Chapter and Verse</string>
   </property>
   <widget class="QComboBox" name="cbStartChapter">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStartVerse">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>65</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chapter:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Verse:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>380</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Ending Chapter and Verse</string>
   </property>
   <widget class="QComboBox" name="cbEndChapter">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbEndVerse">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>65</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chapter:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Verse:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbCopy">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>470</y>
     <width>391</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy To Clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>505</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteVerses">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>35</y>
     <width>401</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>380</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Bible Book</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Book Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBibleBook">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>30</y>
      <width>250</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbAddVerse">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>5</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Verse Numbers</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbAddBookChapter">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>5</y>
     <width>166</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Book And Chapter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
